db-recipe-book/data/migrations/20181011171732_dishes_table.js

```diff
@@ -0,0 +1,18 @@
+
+exports.up = function(knex, Promise) {
+  return knex.schema.createTable('dishes', function(tbl) {
+      tbl.increments();
+      tbl.string('name', 50).notNullable();
+      tbl.integer('dish_id');
```

Comment:
You don't need a `dish_id` column since you already have an `id` column for the dishes table that is automatically created by line 4.